### PR TITLE
[RC2] Fix PDF rendering issues

### PR DESCRIPTION
**Pull request by collivier**
It also formats tables and fixes duplicated title.
It cleans bundle data which are manually removed by GSMA.

### CODE REVIEW

**Comment by pgoyal01** [comment]
Why are we removing section headers? There is something wrong if we can't use section headers because they repeat?

File: doc/ref_cert/RC2/index.rst [hunk 1]
```diff
@@ -1,56 +1,10 @@
-Reference Conformance - Kubernetes based platform for Telecom
-=============================================================
-
-This is the Kubernetes Based Reference Conformance (RC-2)
-
-Release Information
```
Comment (CsatariGergely) [inline doc/ref_cert/RC2/index.rst:6]: Don't we need this information somewhere?

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_cert/RC2/index.rst [hunk 2]
```diff
@@ -1,56 +1,10 @@
...
+=================================================
+Anuket Reference Conformance for Kubernetes (RC2)
+=================================================
+
+.. include:: chapters/chapter01.rst
+.. include:: chapters/chapter02.rst
+.. include:: chapters/chapter03.rst
+.. include:: chapters/chapter04.rst
+.. include:: chapters/chapter05.rst
+.. include:: chapters/chapter06.rst
```
Comment (gkunz) [inline doc/ref_cert/RC2/index.rst:10]: I am generally ok with putting everything onto a single page. However, does this concept also support header numbering which is important in the RM and RAs?

Comment (collivier) [reply doc/ref_cert/RC2/index.rst:10]: yes it works.

File: doc/ref_cert/RC2/index.rst (hunk 1 above)
Comment (collivier) [reply doc/ref_cert/RC2/index.rst:6]: this data is in conf.py. see the PDF rendering

**Comment by collivier** [comment]
@pgoyal01 the bundle histroy, etc. is not part of GSMA and it fully breaks all titles. I can share 2 PDFs to highlight the issues if it helps.

**Review by karinesevilla — APPROVED** [review]

**Comment by collivier** [comment]
@CsatariGergely @gkunz @pgoyal01 I reduced the scope of this PR.
It now only removes the bundle history and converts a few tables.

**Review by gkunz — APPROVED** [review]

**Pull request merged**